tests/unit/test_pool.py

```diff
@@ -304,18 +315,22 @@ def test_spans_pool_bind(self):
         except Exception:
             pass
 
+        span_list = self.get_finished_spans()
+        got_span_names = [span.name for span in span_list]
+        want_span_names = ["testBind", "CloudSpanner.FixedPool.BatchCreateSessions"]
+        assert got_span_names == want_span_names
+
         wantEventNames = [
             "Requesting 1 sessions",
-            "Creating 1 sessions",
```

Comment:
Why is this event removed? As per code this event is still there.
Can you check if it is not getting added?

Reply:
Taht event is no longer on testBind, it'll be on CloudSpanner.FixedPool.BatchCreateSessions @harshachinta.

Reply:
I added tests to check for all the events so that we can see them.